app/models/piece.rb

```diff
@@ -90,6 +90,7 @@ def move_to!(destination_x, destination_y)
     results[:pieces_moved] << piece_moved_hsh
     update(x: destination_x, y: destination_y, first_move: false)
     results
+    Game.change_turn!
```

Comment:
would be lowercase `game` here, to reference the game that the piece belongs to.